src/ol/interaction/translate.js

```diff
@@ -234,6 +240,35 @@ ol.interaction.Translate.prototype.setHitTolerance = function(hitTolerance) {
 
 
 /**
+ * @inheritDoc
+ */
+ol.interaction.Translate.prototype.setMap = function(map) {
+  var oldMap = this.getMap();
+  ol.interaction.Pointer.prototype.setMap.call(this, map);
+  this.updateState_(oldMap);
+};
+
+
+/**
+ * @param {ol.Map} oldMap Old map.
+ * @private
+ */
+ol.interaction.Translate.prototype.updateState_ = function(oldMap) {
```

Comment:
Ok, I did that. But I didn't override `setActive` because someone could change the active status with `.set('active', false);`